eidpdata/src/main/java/uk/ac/ucl/eidp/data/PoolStrategy.java

```diff
@@ -58,8 +58,9 @@
     private StatementGenerator statementGenerator;
     
     private void initialiseConnection() {
-        if (properties.containsKey(SQL_DIALECT_PROP))
+        if (properties.containsKey(SQL_DIALECT_PROP)) {
```

Comment:
\[Checkstyle\] WARNING: 'if' have incorrect indentation level 8, expected level should be 4\.